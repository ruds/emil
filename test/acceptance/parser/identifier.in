abc ; foo.bar.baz ;
op ++ ; op prefix \= ; op prefix foo.\ ; op foo.bar.++ ;
++ ;
foo.\ ;
