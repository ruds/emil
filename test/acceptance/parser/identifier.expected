0001 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             ()
             :identifier
             "616263"
             :is_op
             false
             :is_prefix_op
             false))
0001 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             (
                 "666F6F"
                 "626172"
             )
             :identifier
             "62617A"
             :is_op
             false
             :is_prefix_op
             false))
0002 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             ()
             :identifier
             "2B2B"
             :is_op
             true
             :is_prefix_op
             false))
0002 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             ()
             :identifier
             "5C3D"
             :is_op
             true
             :is_prefix_op
             true))
0002 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             ("666F6F")
             :identifier
             "5C"
             :is_op
             true
             :is_prefix_op
             true))
0002 (ExprTopDecl
         :expr
         (IdentifierExpr
             :qualifiers
             (
                 "666F6F"
                 "626172"
             )
             :identifier
             "2B2B"
             :is_op
             true
             :is_prefix_op
             false))
0003 ERROR
0004 ERROR
0005 (EndOfFileTopDecl)
